envs/example/defaults.yml

```diff
@@ -21,6 +21,7 @@ secrets:
   horizon_secret_key:           asdf
   glance_sync:      ADQ64XUQLUWH75M634RVBLP55RKPGGOWG
   monitor_password: asdf
+  telemetry_password: asdf
```

Comment:
The telemetry_password is used to config the "telemetry_secret" in the ceilometer.conf. When a meter in one ceilometer component is sent to other ceilometer component, this secret needs to be passed with the meter for other component to verify that the meter is coming from a valid source. Dean, please just call it telemetry_secret so it is clear and consistent for what it is. Password will be a little confusing.